modules/apps/style-book/style-book-test/src/testIntegration/java/com/liferay/style/book/service/test/StyleBookEntryServiceTest.java

```diff
@@ -151,16 +194,7 @@ public void testGetStyleBookEntryByExternalReferenceCodeWithoutViewPermission()
 				RandomTestUtil.randomString(), null,
 				RandomTestUtil.randomString(), _serviceContext);
 
-		RoleTestUtil.removeResourcePermission(
-			RoleConstants.GUEST, StyleBookEntry.class.getName(),
-			ResourceConstants.SCOPE_INDIVIDUAL,
-			String.valueOf(styleBookEntry.getStyleBookEntryId()),
-			ActionKeys.VIEW);
-		RoleTestUtil.removeResourcePermission(
-			RoleConstants.SITE_MEMBER, StyleBookEntry.class.getName(),
-			ResourceConstants.SCOPE_INDIVIDUAL,
-			String.valueOf(styleBookEntry.getStyleBookEntryId()),
-			ActionKeys.VIEW);
+		_removeResourcePermissions(styleBookEntry.getStyleBookEntryId());
```

Comment:
Since you extracted this part, it seems `testGetStyleBookEntryByExternalReferenceCodeWithoutViewPermission` also have a duplication. Could you replace it for this new method?

Reply:
Also, please rename this commit. SF does not change logic.

Reply:
@lfbesada Since you're refactoring, could you please create a setup for

```java
StyleBookEntry styleBookEntry =
  _styleBookEntryService.addStyleBookEntry(
    RandomTestUtil.randomString(), _group.getGroupId(),
    RandomTestUtil.randomString(), null,
    RandomTestUtil.randomString(), _serviceContext);
```

and other code duplication please?